desktop/desktop_panels.xml

```diff
@@ -24,46 +24,53 @@
 				<stateframe>
 					<hover name="fieldfocus" offset="7,5,7,5" />
 				</stateframe>
-				<!-- <font>calendarbold</font> -->
 				<readonly />
 				<center />
+				<!-- Opening timedevents, timedreminders, and travelmanager in onInit() allows for their
+				triggers to work correctly on campaign open, otherwise they don't work until opened manually. -->
 				<script>
 					function onInit()
-						if DB.getValue("calendar.newcampaign") == nil or DB.getValue("calendar.newcampaign") == 0 then
-												
-							local nYear = DB.getValue("calendar.current.year", 0);
+						if DB.getValue(TimeManager.CAL_NEWCAMPAIGN) == nil or DB.getValue(TimeManager.CAL_NEWCAMPAIGN) == 0 then
+							local nYear = DB.getValue(TimeManager.CAL_CUR_YEAR, 0);
 							if nYear == nil or nYear == 0 then
-								DB.setValue("calendar.current.year", "number", 1);
+								DB.setValue(TimeManager.CAL_CUR_YEAR, "number", 1);
 							end
-							local nMonth = DB.getValue("calendar.current.month", 0);
+
+							local nMonth = DB.getValue(TimeManager.CAL_CUR_MONTH, 0);
 							if nMonth == nil or nMonth == 0 then
-								DB.setValue("calendar.current.month", "number", 1);
+								DB.setValue(TimeManager.CAL_CUR_MONTH, "number", 1);
 							end
-							local nDay = DB.getValue("calendar.current.day", 0);
+
+							local nDay = DB.getValue(TimeManager.CAL_CUR_DAY, 0);
 							if nDay == nil or nDay == 0 then
-								DB.setValue("calendar.current.day", "number", 1);
+								DB.setValue(TimeManager.CAL_CUR_DAY, "number", 1);
 							end
-							local nMinute = DB.getValue("calendar.current.minute", 0);
+
+							local nMinute = DB.getValue(TimeManager.CAL_CUR_MIN, 0);
 							if nMinute == nil then
-								DB.setValue("calendar.current.minute", "number", 0);
+								DB.setValue(TimeManager.CAL_CUR_MIN, "number", 0);
 							end
-							local nHour = DB.getValue("calendar.current.hour", 0);
+
+							local nHour = DB.getValue(TimeManager.CAL_CUR_HOUR, 0);
 							if nHour == nil then
-								DB.setValue("calendar.current.hour", "number", 0);
+								DB.setValue(TimeManager.CAL_CUR_HOUR, "number", 0);
 							end
+
 							Interface.openWindow("calendar", "DB");
-							DB.setValue("calendar.newcampaign", "number", 1);
+							DB.setValue(TimeManager.CAL_NEWCAMPAIGN, "number", 1);
 						end
-						DB.addHandler("calendar.current.hour", "onUpdate", onSourceChanged);
+
+						Interface.openWindow("timedevents", "DB");
+						Interface.toggleWindow("timedevents", "DB")
+						Interface.openWindow("timedreminders", "DB");
+						Interface.toggleWindow("timedreminders", "DB")
+						Interface.openWindow("travelmanager", "DB");
+						Interface.toggleWindow("travelmanager", "DB")
+
```

Comment:
This was the key to making the events and reminders work on open.  They were never opened so they couldn't be loaded/accessed.  This quick toggle solves that and makes the events/reminders ready immediately.